test/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ if (GO_BIN)
     add_custom_command(
         OUTPUT ${GoFFI_TestBin}
         COMMAND ${CMAKE_COMMAND} -E env CGO_LDFLAGS=-L${LibPath} ${GO_BIN} ARGS build -o ${GoFFI_TestBin} ${GoFFI_Src}
-        DEPENDS ${GoFFI_Src}
+        DEPENDS ${GoFFI_Src} sysfail
```

Comment:
Interesting, looks like compiler actually wants the shared-lib. Does it avoid hardcoding absolute path if not given? Or does it simply fail?

Reply:
It fails with the below error

```
[ 20%] Building ffi.go
# command-line-arguments
/usr/lib/go-1.22/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: cannot find -lsysfail: No such file or directory
collect2: error: ld returned 1 exit status
```